docs/source/configuration.rst

```diff
@@ -99,4 +99,52 @@ RESPONSE_VALIDATION
     that they conform to the protocol. This should only be used for development
     purposes.
 
+OIDC_PROVIDER
+    If this value is provided, then OIDC is configured and SSL is used. It is
+    the URI of the OpenID Connect provider, which should return an OIDC
+    provider configuration document.
 
+SERVER_NAME
+    A standard flask configuration variable, but required if OIDC is configured
+    so the server can know its correct redirect URI.
+
+OIDC_CLIENT_ID, OIDC_CLIENT_SECRET
+    These are the client id and secret arranged with the OIDC provider,
+    if client registration is manual (google, for instance). If the provider
+    supports automated registration they are not required or used.
```

Comment:
I don't have an informed opinion here, so I'd go with @bartgrantham's views on this. Do we lose any significant functionality by removing support for dynamic registration? I.e., are there any cases in which we _must_ use dynamic registration?